packages/react/src/patterns/blocks/ContentGroupCards/__stories__/index.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../../styles/scss/patterns/blocks/contentgroupcards/index';
+
+.#{$prefix}--contentgroupcards .#{$prefix}--content-group__children {
```

Comment:
minor thing, wondering if we should make the naming consistent with the `bx--content-group` class name, and have `bx--content-group-cards`

Reply:
This was changed in the last commit, thanks!